dev/user_repl.clj

```diff
@@ -61,7 +62,8 @@
                request-handler-service
```

Comment:
I'd also comment out / remove the `:invalid-in-puppet-4` route from the `master-service` route definition in the `puppet-server-conf` function.